wallet/src/main/scala/org/bitcoins/wallet/DLCWallet.scala

```diff
@@ -516,11 +558,12 @@ abstract class DLCWallet extends LockedWallet with UnlockedWalletApi {
   override def acceptDLCMutualClose(
       mutualCloseSig: DLCMutualCloseSig): Future[Transaction] = {
     for {
-      _ <- updateDLCOracleSig(mutualCloseSig.eventId, mutualCloseSig.oracleSig)
+      dlcDb <- updateDLCOracleSig(mutualCloseSig.eventId,
+                                  mutualCloseSig.oracleSig)
+      (_, dlcOffer, dlcAccept, fundingInputs) <- getAllDLCData(
```

Comment:
was there a reason to use the older value rather than the newer one for `dlcDb`?

Reply:
No, but I think it makes more sense to get it from `getAllDLCData` so I'll switch it